pillar/heroku/ocw-studio.sls

```diff
@@ -6,26 +6,36 @@
       'app_name': 'ocw-studio-ci',
       'env_name': 'ci',
       'MAILGUN_SENDER_DOMAIN': 'ocw-ci.mail.odl.mit.edu',
+      'OCW_STUDIO_LOG_LEVEL': 'INFO',
+      'sentry_log_level': 'WARN',
       'SITE_NAME': 'MIT OCW Studio CI',
       'vault_env_path': 'rc-apps'
       },
     'rc': {
       'app_name': 'ocw-studio-rc',
+      'env': 'qa',
       'env_name': 'rc',
       'MAILGUN_SENDER_DOMAIN': 'ocw-rc.mail.odl.mit.edu',
+      'OCW_STUDIO_LOG_LEVEL': 'INFO',
+      'sentry_log_level': 'WARN',
       'SITE_NAME': 'MIT OCW Studio RC',
       'vault_env_path': 'rc-apps'
       },
     'production': {
       'app_name': 'ocw-studio',
+      'env': 'production',
       'env_name': 'production',
       'MAILGUN_SENDER_DOMAIN': 'ocw.mail.odl.mit.edu',
+      'OCW_STUDIO_LOG_LEVEL': 'INFO',
+      'sentry_log_level': 'WARN',
       'SITE_NAME': 'MIT OCW Studio',
-      'vault_env_path': 'production-apps'
+      'vault_env_path': 'production-apps',
       }
 } %}
 {% set env_data = env_dict[environment] %}
 {% set business_unit = 'ocw' %}
+{% set pg_creds = salt.vault.cached_read('postgres-ocw-studio-applications-{}/creds/ocw_studio'.format(env_data.env), cache_prefix='heroku-ocw-studio') %}
```

Comment:
This will need to go under the conditional so that it doesn't error on pillar rendering due to the mount not existing in CI.